.changeset/@mastra_core-9806-dependencies.md

```
@@ -0,0 +1,5 @@
+---
+"@mastra/core": patch
+---
+dependencies updates:
+  - Updated dependency [`@modelcontextprotocol/sdk@^1.20.2` ↗︎](https://www.npmjs.com/package/@modelcontextprotocol/sdk/v/1.20.2) (from `^1.17.5`, in `dependencies`)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)